common/src/main/java/dev/ftb/mods/ftbchunks/api/client/minimap/MinimapContext.java

```diff
@@ -0,0 +1,39 @@
+package dev.ftb.mods.ftbchunks.api.client.minimap;
+
+import dev.ftb.mods.ftbchunks.client.map.MapDimension;
+import dev.ftb.mods.ftblibrary.math.XZ;
+import net.minecraft.client.Minecraft;
+import net.minecraft.client.player.LocalPlayer;
+
+import java.util.Map;
+
+/**
+ * Minimal context for Minimap Info Components
+ *
+ * @param minecraft The Minecraft instance (Helper)
+ * @param player The client player
+ * @param mapDimension The dimension of the players location
+ * @param region The region of the players location
```

Comment:
Is it necessary to store the chunk X/Z _and_ the region?  One can be trivially calculated from the other.

Also, could the player X/Y/Z just be stored as a `Vec3`?